ch_util/rfi.py

```diff
@@ -104,7 +112,7 @@ def flag_dataset(
 
     # Apply the frequency cut to the data (add here because we are distributed
     # over products and its easy)
-    freq_mask = frequency_mask(data.freq)
+    freq_mask = frequency_mask(data.freq, data.time[0])
```

Comment:
I personally like to always give the name for keyword arguments, e.g.
```suggestion
    freq_mask = frequency_mask(data.freq, timestamp=data.time[0])
```
This is somewhat optional, so pick your own preferred style, but it's probably clearer and easier with long argument lists where you don't necessarily remember the order.